guides/getting-started.md

```diff
@@ -34,9 +34,7 @@ yarn add angular/material2-builds angular/cdk-builds
 
 ### Step 2: Animations
 
-Some Material components depend on the Angular animations module in order to be able to do
-more advanced transitions. If you want these animations to work in your app, you have to
-install the `@angular/animations` module and include the `BrowserAnimationsModule` in your app.
+Animations package is required to build application with Material components, but you don't have to configure it unless you use animated components like Drawer, Bottom Sheet etc.
```

Comment:
nit: "Material components" -> "Angular Material"

Reply:
Another nit: "Animations package" -> "The `@angular/animations` package"